examples/dataproc-gcs-connector/build_gcs_connector.sh

```diff
@@ -0,0 +1,80 @@
+#!/bin/bash
```

Comment:
run [shellcheck](https://www.shellcheck.net/)
```
Line 37:
cd hadoop-connectors
^-- SC2164: Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

Did you mean: (apply this, apply all SC2164)
cd hadoop-connectors || exit
 
Line 50:
if [[ "$?" -ne 0 ]] ; then
      ^-- SC2181: Check exit code directly with e.g. 'if mycmd;', not indirectly with $?.
 
Line 52:
  exit $rc
       ^-- SC2154: rc is referenced but not assigned.
       ^-- SC2086: Double quote to prevent globbing and word splitting.

Did you mean: (apply this, apply all SC2086)
  exit "$rc"
 
Line 55:
cd ..
^-- SC2103: Use a ( subshell ) to avoid having to cd back.
 
Line 58:
cd terraform
^-- SC2164: Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

Did you mean: (apply this, apply all SC2164)
cd terraform || exit
```